scripts/fast-reboot

```diff
@@ -513,6 +513,12 @@ if [ -x /sbin/hwclock ]; then
     /sbin/hwclock -w || /bin/true
 fi
 
+# Enable Watchdog Timer
+if [[ -e /usr/local/bin/platform_watchdog_enable.sh ]]; then
+    debug "Enabling Watchdog before fast-reboot"
+    /usr/local/bin/platform_watchdog_enable.sh
```

Comment:
Please don't call Dell watchdog script from here. Call a secondary script to call different script according to the platform.